### PR TITLE
Problem (Fix #196): details of TDBE usage of light client are missing

### LINKED ISSUE #196: Problem: details of TDBE usage of light client are missing
as mentioned in the related implementation issue:
> In particular, as the code runs inside enclaves, 1) it's trusted (-> needs to be simple and small without too many dependencies), 2) there's no "trusted" persistent storage (in particular, the host may replay the old state.

> With regards to "light client state" / trusted anchor, it implies two parts:

> 1) (no state) the full genesis.json isn't known before TDBE is compiled: https://github.com/crypto-com/chain-docs/blob/master/docs/modules/tdbe.md#genesisjson-creation -- but some parts of it should be known and should be compile-time parameters https://github.com/crypto-com/chain/issues/1996 that are cross-checked
> 2) once a node "successfully" joins (either valid genesis.json with related mls/keypackage payload OR valid committed nodejointx) OR updates (successful committed SelfUpdateProposal after NACK timeout), it can "seal" its identity+HPKE keys altogether with the "light client state" at that point

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- add light client verification steps

### CODE REVIEW

File: docs/modules/tdbe.md [hunk 1]
```diff
@@ -32,16 +32,13 @@ have to follow below procedure to successfully join the network:
...
   by the node operator (see below for mode details on **Transaction signing**).
 - If the `NodeJoinTx` is accepted in a block, new node can start to operate normally (as `Commit` with Add-only proposal
   does not need to fill paths -- node need to wait for NACK timeout).
   - New node's TDBE server can apply `Welcome` message to update its state.
-  - If one wishes to join as the "council node" (have validator capabilities and responsibilities): once the local node
-  catches up fully with the latest state (i.e. chain-abci gets to the latest block), the operator can submit
-  the second `NodeJoinTx` transaction with its node's consensus public key 
 - If the `NodeJoinTx` is not accepted (this can happen when there was another `Commit` in the meantime), then restart
-  the process.
+  the proocess.
```
Comment (leejw51crypto) [inline docs/modules/tdbe.md]: the process?

**Review by leejw51crypto — APPROVED** [review]
lgtm

File: docs/modules/tdbe.md [hunk 2]
```diff
@@ -284,14 +330,14 @@ with council node and community node variants.
...
-  CommunityToCouncilNodeSwitch,
   // KeyPackage
   Genesis(Vec<u8>),
-  CommunityNodeJoin {
+  NodeJoin {
     add: Vec<u8>, // MLSPlaintext -- Add
     commit: Vec<u8> // MLSPlaintext -- Commit
 }
 ```
+TODO: extra rules for CouncilNode -> CommunityNode (needs to unbond / become inactive first?)
```
Comment (tomtau) [inline docs/modules/tdbe.md]: this reverts the changes/recent revision

File: docs/modules/tdbe.md [hunk 3]
```diff
@@ -32,16 +32,13 @@ have to follow below procedure to successfully join the network:
   - New node, after receiving a response from remote TDBE server, verifies (TODO: Define validations on TDBE response)
     it and starts the whole process again if the verification fails (with a different remote TDBE server). If the
     verification is successfull, it continues to the next step.
-- New node constructs `NodeJoinTx` (for community node) and submits the transaction after getting it signed
+- New node constructs `NodeJoinTx` (for council or community node) and submits the transaction after getting it signed
```
Comment (tomtau) [inline docs/modules/tdbe.md]: only for community node -- then there'll be another node join after the sync up if one wishes to continue as council node

**Review by tomtau — CHANGES_REQUESTED** [review]
looks good, but it reverted some changes from https://github.com/crypto-com/chain-docs/pull/217/files

File: docs/modules/tdbe.md (hunk 3 above)
Comment (yihuang) [reply docs/modules/tdbe.md]: yes, it's by accident, fixed.

File: docs/modules/tdbe.md (hunk 2 above)
Comment (yihuang) [reply docs/modules/tdbe.md]: it's by accident, fixed.

File: docs/modules/tdbe.md (hunk 1 above)
Comment (yihuang) [reply docs/modules/tdbe.md]: fixed

**Review by tomtau — APPROVED** [review]

**Pull request merged**